main.tf

```diff
@@ -107,7 +107,9 @@ resource "aws_lb" "main" {
 }
 
 resource "aws_lb_target_group" "https" {
-  name        = "ecs-${var.name}-${var.environment}-https"
+  # Name must be less than 32 characters, or AWS API returns error.
+  # Error: "name" cannot be longer than 32 characters
+  name        = "${var.name}-${var.environment}-https"
```

Comment:
Does changing the name result in a replacement of the target group?

Reply:
Yeah, probably good idea to try to keep same if we can.  I added a `target_group_name` to override this instead of changing default.